cmd/crdvalidator/annotation/annotation.go

```diff
@@ -0,0 +1,16 @@
+package annotation
+
+import (
+	"github.com/operator-framework/rukpak/cmd/crdvalidator/handlers"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func Add(obj client.Object) {
+	// Add the annotation that crdvalidator is looking for
+	annotations := obj.GetAnnotations()
+	if annotations == nil {
+		annotations = map[string]string{}
+	}
+	annotations[handlers.ValidateAnnotation] = "true"
```

Comment:
nit: I think `"enabled"` makes a bit more sense than `"true"` in this context? Since we are specifying that the webhook should be active for this particular CRD. WDYT?